src/mon/MonMap.cc

```diff
@@ -357,7 +357,7 @@ void MonMap::print_summary(ostream& out) const
     out << p->first << "=" << p->second.public_addrs;
     has_printed = true;
   }
-  out << "}";
+  out << "}" << " removed_ranks: {" << removed_ranks << "}";
```

Comment:
I'd expect the addition of this log line to break some unit tests, which probably need to get their output-text-matching files updated to match the new format.

Reply:
I see, I'll try to find a way to output the `removed_ranks` better

Reply:
This is the right way. I think you just need to update some of the out.t test files.

Or maybe we don't actually check the Mon map output in any of those? Which I would be surprised about...

...maybe those tests all use the dump() output instead? Actually we should probably put this there instead of just in print_summary? Would be a bit weird for the summary to have info not in the dump. :)

Reply:
make sense